README.md

```diff
@@ -1,3 +1,21 @@
+# Pants setup
+
 This repository contains the bootstrap needed to get you up and running with pants.
 
 Follow the instructions at https://www.pantsbuild.org/install.
+
+### Development
+
+We use [tox](https://testrun.org/tox/en/latest/) for test automation. To run the test suite and linters, invoke tox:
+
+```bash
+$ tox
+```
+
+You may run certain environments with `tox -e` (run `tox -a` to see all options):
```

Comment:
```suggestion
You may run certain checks (`environments` in tox) with `tox -e` (run `tox -a` to see all options):
```

The fact that they call them `environments` but they actually _run_ the thing was a super big gotcha for me when I touched tox for the first time.

Reply:
Good suggestion!

FWIT, I think the word `environments` comes from the fact that one of the main features of Tox is to allow testing your library with different versions of Python, e.g. `py27` vs. `py36`. These are distinct "environments". 

Still, confusing for first-time users and I like how your change uses familiar language while still clarifying what the official Tox verbiage is.